src/devEnvs.ts

```diff
@@ -306,10 +325,31 @@ export async function configureDemistoVars(): Promise<void> {
         env["DEMISTO_API_KEY"] = apiKey;
       }
     });
+  
+  // XSIAM Auth ID
+  if (configuredPlatform === Platform.XSOAR8_XSIAM){
+    await vscode.window
+      .showInputBox({
+        title: "XSIAM Auth ID",
+        ignoreFocusOut: true,
```

Comment:
can you add ignorefocusout to all input boxes here?

Reply:
done